.gitignore

```diff
@@ -20,3 +20,11 @@ yarn-debug.log*
 yarn-error.log*
 
 Session.vim
+
+# yarn
+.yarn/
+!.yarn/patches
+!.yarn/releases
+!.yarn/plugins
+!.yarn/sdks
+!.yarn/versions
```

Comment:
Hm... we should ignore these too, no? We don't want to track anything in the yarn dir.

Reply:
This is the recommended gitignore for yarn3 since it allows us to keep our version information in sync.